larray_editor/comparator.py

```diff
@@ -133,11 +134,11 @@ def update_isequal(self):
                 # scale reldiff to range 0-1 with 0.5 for reldiff = 0
                 self.bg_value = (reldiff / maxabsreldiff) / 2 + 0.5
             else:
-                self.bg_value = full_like(self.array, 0.5)
+                self.bg_value = la.full_like(self.array, 0.5)
         except TypeError:
             # str/object array
-            maxabsreldiff = nan
-            self.bg_value = full_like(self.array, 0.5)
+            maxabsreldiff = np.nan
```

Comment:
use la.nan please :)

Reply:
What's the difference?

Reply:
None at this point, but it could in the future. Since we provide it in the larray namespace, it's better to use it.